storage/new_bucket/main.tf

```diff
@@ -32,14 +32,12 @@ resource "google_storage_bucket" "static" {
 # [END storage_create_new_bucket_tf]
 
 # [START storage_upload_object_tf]
-# Upload files
-# Discussion about using tf to upload a large number of objects
-# https://stackoverflow.com/questions/68455132/terraform-copy-multiple-files-to-bucket-at-the-same-time-bucket-creation
+# Creates a text file based on content_type with content
```

Comment:
Can we change this line to simply "Upload a file as an object to a bucket"? This would make the comment more consistent with the heading of the doc ("Upload an object to a bucket").

Reply:
it's possible and could be done in a separate new sample PR. This PR is meant to resolve the issue with the comments as they are.